otherlibs/stdune/src/path.ml

```diff
@@ -1469,6 +1472,11 @@ let drop_prefix_exn t ~prefix =
   | Some p -> p
 ;;
 
+let is_broken_symlink = function
+  | External e -> External.is_broken_symlink e
+  | _ -> false
```

Comment:
Is this actually correct? Can't other path variants have broken symlinks? It's possible that they cannot, but reading this casually this question pops up on my mind. Thus it would probably be better to either implement the check on other types as well, throw an error or have a comment why this is correct.

Reply:
I'll add a comment.